### DISCUSSION TITLE
Request: Allow MySQL password to be specified interactively

### DISCUSSION THREAD

**Discussion by Nevario** [amacneil/dbmate#353] [Feature Ideas]
Firstly, thank-you for an awesome tool!

For the ultra paranoid of us, who prefer not to leave the MySQL password in an env file (even on a well secured Bastion host), it would be awesome if we could omit the password and specify it interactively with something like a `-p` flag. 

This is how we use currently use Skeema.io which allows this for this workflow.

Please close this request if it's not feasible, but just thought I would mention it... just in case :)

**Comment by amacneil** [comment]
I think this is a good idea (we should support it for all database drivers, not just MySQL). Open to a PR.
